docker/docker-compose-dev.yaml

```diff
@@ -0,0 +1,35 @@
+# docker-compose -f ./docker/docker-compose-dev.yaml up --build -d
+# docker-compose -f ./docker/docker-compose-dev.yaml down
+
+version: "2.4"
+x-net: &net
+  networks:
+      - main
+services:
+  db:
```

Comment:
```suggestion
  calcium-imaging-dev-db:
```

Reply:
I have experienced collisions when the `db` container for different workflows has the same name.